contents/tutorials/frontend-vs-backend-group-analytics.md

```diff
@@ -0,0 +1,156 @@
+    score: 99,
+})
+```
+
+Because sessions connect users and groups, you need to call `posthog.reset()` when a user changes (logs out) to ensure events aren’t captured as belonging to that user and group. If you want to connect the user back to a group, you’ll need to call `posthog.group()` again.
+
+## Backend group analytics implementation
+
+The backend implementation is different. Backend libraries have no concept of a session (they aren’t stateful). This means they don’t get the abstraction of combining the user (person) and group. Every time you capture an event, you must add the group data to it.
+
+<MultiLanguage>
+
+```python
+posthog.capture('ian@posthog.com', 'login', groups={'company': 'posthog'})
+```
+
+```go
+client.Enqueue(posthog.Capture{
+    DistinctId: "ian@posthog.com",
+    Event:      "login",
+    Groups: posthog.NewGroups().
+        Set("company", "posthog").
+})
+```
+
+```node
+posthog.capture({
+    event: "login",
+    distinctId: 'ian@posthog.com',
+    groups: { company: 'posthog' }
+})
+```
+
+```php
+PostHog::capture(array(
+    'distinctId' => 'ian@posthog.com',
+    'event' => 'login',
+    '$groups' => array("company" => "posthog")
+));
+```
+
+```segment
+analytics.track('login', {
+    "$groups": {
+        "company": "posthog"
+    }
+})
+```
+
+</MultiLanguage>
+
+You also can’t update the group properties from the capture call. You must use a separate `group_identify` which looks like this:
+
+<MultiLanguage>
+
+```python
+posthog.group_identify('company', 'posthog', {
+    'company_name': 'PostHog',
+    'score': 99
+})
+```
+
+```go
+client.Enqueue(posthog.GroupIdentify{
+    Type: "company",
+    Key:  "posthog",
+    Properties: posthog.NewProperties().
+        Set("company_name", "PostHog").
+        Set("score", 99),
+})
+```
+
+```node
+posthog.groupIdentify({
+    groupType: 'company',
+    groupKey: 'posthog',
+    properties: {
+        company_name: 'PostHog',
+        score: 99
+    }
+})
+```
+
+```php
+PostHog::groupIdentify(array(
+    'groupType' => 'company',
+    'groupKey' => 'posthog',
+    'properties' => array("company_name" => "PostHog", "score" => 99)
+));
+```
+
+```segment
+analytics.track('$groupidentify', {
+    "$group_type": "company",
+    "$group_key": "posthog",
+    "$group_set": {
+        "company_name": "PostHog",
+        "score": 99
+    }
+})
+```
+
+</MultiLanguage>
+
+PostHog currently stores groups separately from events, but soon, we will merge them. This means calling `group_identify` will not update the group details for past events (like it does now). To handle this, we recommend calling `group_identify` whenever group properties change. This makes sure events always have the most updated group properties.
+
+The benefit of the backend is that you don’t have to call `reset()` when you’re done because you have no session to reset. If you’ve set up your capture calls properly, they should be sending data from the active user or group (which will be none).
```

Comment:
Optional whether you want to include or not, just more context:

There are no sessions on the backend because usually backends deal with multiple requests from different people one after the other. Thus it becomes very awkward to enforce creating a session that lasts only for one request, and reduces flexibility for implementers.

Reply:
added, thanks!